cfme/test_framework/config.py

```diff
@@ -0,0 +1,53 @@
+import os
+import warnings
+import yaycl
+
+
+class Configuration(object):
+    def __init__(self):
+        self.yaycl_config = None
+
+    def configure(self, config_dir, crypt_key_file=None):
```

Comment:
Documentation comment please with explanation how this shall be used.

And if we can also pass just a string, without file (eg. as a parameter to py.test), that would be awesome.